tendenci/apps/forms_builder/forms/forms.py

```diff
@@ -258,6 +258,11 @@ def email_to(self):
             field_class = field.field_type.split("/")[0]
             if field_class == "EmailVerificationField":
                 return self.cleaned_data["field_%s" % field.id]
+            
+        user_email = getattr(self.user, "email", None).strip()
```

Comment:
In case of anonymous user, `getattr(self.user, "email", None)` would return `None`.  It would then throw an AttributeError error because `None` is not a string.

Reply:
So: `user_email = getattr(self.user, "email", "").strip()` is preferred?

Reply:
Yes, that would be good.

Reply:
WIll fix and push to branch.